### PR TITLE
Redshift: support 3 part id

### PR DESCRIPTION
**Pull request by dlawin**
per our slack convo

### CODE REVIEW

**Comment by erezsh** [comment]
PR name should mention it's for Redshift.

**Comment by dlawin** [comment]
> PR name should mention it's for Redshift.

added

**Comment by dlawin** [comment]
I'll take a deeper look at this on Monday. It looks like 3 part ids are supported for redshift but I could do more testing.

And if we are using 3 part, should use database in queries

**Comment by dlawin** [comment]
Do I need to extend the URI params?

File: sqeleton/databases/redshift.py [hunk 1]
```diff
@@ -98,3 +109,15 @@ def query_table_schema(self, path: DbPath) -> Dict[str, tuple]:
             return super().query_table_schema(path)
         except RuntimeError:
             return self.query_external_table_schema(path)
+
+    def _normalize_table_path(self, path: DbPath) -> DbPath:
```
Comment (dlawin) [inline sqeleton/databases/redshift.py:113]: @erezsh what do you think about: instead of overriding these methods, having a 3 part method in the base class

Comment (erezsh) [reply sqeleton/databases/redshift.py:113]: Yes, I think that's a good idea.

But keep in mind some databases are still only 2-part (or potentially 1-part), so we need to support all these variations.

Comment (dlawin) [reply sqeleton/databases/redshift.py:113]: I will leave this change as is, but I can work on an additional PR to add a base class method + update the 3 part dbs to use that.

**Comment by dlawin** [comment]
@erezsh Are you good with this change? Any tests I can add to ease concerns of a regression?

**Comment by erezsh** [comment]
The tests didn't run on redshift yet. We need to set it back up.

**Comment by dlawin** [comment]
> The tests didn't run on redshift yet. We need to set it back up.

Oh is the URI null or something? Semi-related it'd be nice to have admin on this repo, even to just look at settings and stuff

**Review by erezsh — APPROVED** [review]

**Comment by erezsh** [comment]
LGTM

**Comment by erezsh** [comment]
As for admin privs, that's not up to me. But if we add everyone I'm worried it might turn into a "too many cooks" situation.

**Pull request merged**